components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/consent/ConsentMgtPostAuthnHandler.java

```diff
@@ -246,7 +248,6 @@ private String getSPTenantDomain(ServiceProvider serviceProvider) {
         Set<String> consentClaims = getUniqueLocalClaims(requestedClaims, mandatoryClaims);
```

Comment:
Do we need to add mandatoryClaims now? We shouldn't add ?

Reply:
Yes. It's not required to add. Fixed https://github.com/wso2/carbon-identity-framework/pull/3413/commits/21d2c876c2af1be806c780d1099daeb16466692b